### ISSUE TITLE
[Bug Report] v-select missing "lazy" option

### ISSUE THREAD

**Issue by gabaum10** [vuetifyjs/vuetifyjs.com#616]
### Versions and Environment
**Vuetify:** 1.2.5
**Vue:** 2.5.17
**Browsers:** Chrome 68.0.3440.106
**OS:** Linux x86_64

### Steps to reproduce
Notice in the link above it doesn't inject the select "menu" item until it's clicked.  I see v-select inherits from v-menu.  Personally, I don't see the point to make really any menu non-lazy, but the documentation on the site doesn't mention you can make the v-select lazy.  It does indeed work, it's just missing the doc.  Perhaps it's just a matter of saying "v-select" inherits "x" props from v-menu, or something like that.

### Expected Behavior
N/a

### Actual Behavior
N/a

### Reproduction Link
[https://codepen.io/gabaum10/pen/JmoPmG](https://codepen.io/gabaum10/pen/JmoPmG)

**Comment by jacekkarczmarczyk** [comment]
v-select has `menu-props` prop where you can put `lazy` and other menu props

**Comment by gabaum10** [comment]
Yeah, I discovered that awhile ago.  However, I didn't see any documentation on the site, which is why I left this open.  I haven't checked if it's there or not.
